x11-plugins/*: drop old

mm1ke opened the pull request:
@voyageur 
I think we can start removing the older ebuilds from the tree. EAPI6/7 ebuilds are already stable for a few months now ;)
BTW, there are still a few packages which miss a stable sparc keyword. I guess this is intentionell (isn't sparc only testing anymore?). If yes, we can delete another few ebuilds from the tree, which would mean that those package would go from stable into testing on sparc..

voyageur commented:
Thanks for the PR @mm1ke , I will take a closer look at it next week. Looks good from a quick check

And yes sparc is an exp profile now (you can quickly check them in the Arches dropdown list in bugzilla), so you can add these additional cleanups.
repoman also knows arches status, so if it does not complain when removing last stable version for an arch, that means you can go for it :)

mm1ke commented:
@voyageur 
Thanks for the Information. I was searching to find out about the status of sparc but didn't find anything. Good to know to just look at the bugs :) Also didn't know that repoman checks this too. Nice to know.
I've updated the PR now and added a few other cleanups. Repoman didn't complain so it should be fine.

voyageur commented:
OK, all the PR should be in!
Thanks again @mm1ke repoman should be a bit happier in x11-plugins now
